plugins/kotlin/j2k/shared/tests/testData/newJ2k/visibility/overrideEqualsBuiltIn.java

```diff
@@ -0,0 +1,9 @@
+// IGNORE_K2
+import org.jetbrains.annotations.Nullable;
+
+public class Test {
+    @Override
+    public Boolean equals(@Nullable Any other) {
```

Comment:
Please don't use Kotlin `Any` in Java tests. The test should use types that are natural to use in Java code.